advanced/am-pattern-4/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@chathuraswso2 please update the `CHANGELOG.md`, main README documentation regarding the new release.